constants/task_groups.json

```diff
@@ -83,7 +83,14 @@
         "Scene Generation",
         "Song Generation",
         "Story Arc Construction",
-        "Taxpayer Receipt Generation"
+        "Taxpayer Receipt Generation",
+        "Travel and Leisure",
+        "Environmental Issues",
+        "Business and Marketing",
+        "Legal and Government",
+        "Health and Wellness",
+        "Science and Technology",
+        "Software Development"
```

Comment:
I'm not sure these are tasks, rather than topics?

E.g. software development --> "code generation"?

Reply:
Thank you for clarifying this, I have changed the task corresponding to the original dataset